microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/jaxrs/JaxRsUtils.java

```diff
@@ -66,6 +66,19 @@ public static String getJaxRsPathValue(IAnnotatable annotatable) throws JavaMode
 	public static boolean isJaxRsRequestMethod(IMethod method) throws JavaModelException {
 		return hasAnnotation(method, JAVAX_WS_RS_GET_ANNOTATION);
 	}
+	
+	public static CodeLens createURLCodeLens(String baseURL, String route, String rootPath, String openURICommandId, IMethod method,

```

Comment:
Don't change this method by adding route as parameter. The route should be included inside baseURL.